schedule/yast/btrfs/btrfs_sle_libstorage-ng@ipmi.yaml

```diff
@@ -32,7 +32,6 @@ schedule:
   - installation/first_boot
   - console/system_prepare
   - console/force_scheduled_tasks
-  - console/check_resume
```

Comment:
Excluded for ipmi as I could not manage to have a working VR. Not sure hibernation would work on ipmi (or hyperv) but we don't really validate it anyway, as we only check if the parameter is present, we don't actually try to resume after hibernation.

Reply:
According to [bsc#1188731](https://bugzilla.suse.com/show_bug.cgi?id=1188731):

```
- Hibernation is proposed by default in x86_64 
- Hibernation is NOT proposed for any other architecture
```

So, it seems like 'resume=' parameter should be present for ipmi.

Reply:
I can add the check for ipmi, but that will be without VR then.